resources/clients/react/elwing/src/plugins/item-tracker/ItemTrackerStore.ts

```diff
@@ -37,7 +37,8 @@ export const useItemTrackerState = create<
       actions.clearError();
       actions.setLoading(true);
       try {
-        const items = await workItemService.list({ archive: get().status });
+        const status = get().status;
+        const items = await workItemService.list(status ? { status } : {});
```

Comment:
I find this confusing, because there's a field named "status" that's just a free text field, but this is actually meant to query for whether the "archive" field is set to "active" or "archived".